src/Microsoft.Diagnostics.Monitoring.WebApi/Operation/EgressOperationStore.cs

```diff
@@ -212,7 +223,8 @@ public IEnumerable<Models.OperationSummary> GetOperations(ProcessKey? processKey
                                 Name = processInfo.ProcessName,
                                 ProcessId = processInfo.ProcessId,
                                 Uid = processInfo.RuntimeInstanceCookie
-                            } : null
```

Comment:
Just to be clear, this allows us to specify Process, tag, or both correct?

Reply:
Correct - you can now narrow down your query by specifying one (or both) of them